assets/templates/error.html

```diff
@@ -17,7 +17,7 @@
         </header>
         <div class="container">
           <h1>{{if .ErrorTitle}}{{t .ErrorTitle}}{{else}}{{t "Error Title"}}{{end}}</h1>
-          <footer>
+          <content>
```

Comment:
Is `<content>` an official html tag? I don't see it on https://w3c.github.io/html-reference/elements.html#elements

Reply:
Oh right ! O_O
When I saw it I assumed it was some templating keyword…
You're absolutely right, `<div>` is good enough.